convert2rhel/toolopts.py

```diff
@@ -169,6 +169,11 @@ def _register_parent_options(self, parser):
             action="store_true",
             help="Print traceback in case of an abnormal exit and messages that could help find an issue.",
         )
+        parser.add_argument(
+            "-y",
+            help="Answer yes to all yes/no questions the tool asks.",
+            action="store_true",
+        )
```

Comment:
It is a minor addition, but it was not in the scope of the task. 

Do you see any reason to why the `-y` should be in the parent parser rather than the shared parser?